src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -406,6 +407,7 @@ public VoldemortConfig(Props props) {
 
         this.modifiedProtocol = props.getString("readonly.modify.protocol", null);
         this.modifiedPort = props.getInt("readonly.modify.port", -1);
+        this.bouncyCastleEnabled = props.getBoolean("readonly.bouncy.castle.enabled", false);
```

Comment:
I'm not sure it's a good idea to alter the state of the JVM from within the config class. I would rather leave that in the HdfsFetcher for now. And if/when we want to introduce SSL in Voldemort, we can move that initialization code elsewhere, if need be.

I'm fine with either name. Your call.